src/services/userServices.js

```diff
@@ -0,0 +1,21 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap').factory('logIn', [
+  function login(email,password) {
+    $http({
```

Comment:
Remember that $http object has functions regarding REST methods, so you can use it as a shorthand if you like